docs/UserGuide.md

```diff
@@ -39,21 +39,34 @@ StaffConnect (SC) is a **desktop app for managing Professors' and Tutors' contac
 
 --------------------------------------------------------------------------------------------------------------------
 
-
 ## Navigability
 
 ### Overall UI layout
-<br>![Overall Layout](images/OverallLayout.png)
+<br>![UI General](images/UiAnnotationsGeneraL.png)
+
+#### [UI] Persons Info
+<br>![UI General](images/UiAnnotationsAttributes.png)
+*Note that the availability will not be displayed in sequential order.
+* The values of `Module` will be displayed and stored in capital letters.
+* The values of [`Faculty`](#valid-faculty-values) will be stored to their full names.
+* The values of `day` part of `Availability` will be stored as their full form such as `MONDAY`, `TUESDAY`, `WEDNESDAY`, `THURSDAY`, `FRIDAY`, `SATURDAY`, `SUNDAY`.
+
+#### [UI] Meeting List
+<br>![UI General](images/UiAnnotationsMeetingList.png)
 
+#### [UI] Scrollbars and Menu
+<br>![UI General](images/UiAnnotationsScrollbarsAndMenu.png)
```

Comment:
You are missing the annotations for the horizontal scroll bars at the divider. There are two of them for the pane above and below. You may have to input a much longer meeting description and name to get the horizontal scroll bars to pop out. (and/or resize the window to the smallest possible size)

Reply:
The horizontal scroll bar for person list too , when the name is really long

Reply:
Will amend

Reply:
Amended